hnn_core/cell_response.py

```diff
@@ -423,8 +456,40 @@ def write(self, fname):
                         int(self._spike_gids[trial_idx][spike_idx]),
                         self._spike_types[trial_idx][spike_idx]))
 
+    def write(self, fname, overwrite=True):
+        """Write spiking activity to txt or hdf5 files based on extension.
 
-def read_spikes(fname, gid_ranges=None):
+        Parameters
+        ----------
+        fname : str/Path object
```

Comment:
```suggestion
        fname : str | Path object
```

This is the standard way of documenting when multiple types are possible